packages/gatsby-source-drupal/src/gatsby-node.ts

```diff
@@ -402,6 +401,7 @@ ${JSON.stringify(webhookBody, null, 4)}`
                 createContentDigest,
                 getNode,
                 reporter,
+                pluginOptions,
```

Comment:
I didn't use`getOptions` because I don't think it would work with multiple instances of the plugin. I'll admit I didn't try, but it looks like they would be accessing the same instance of `options`

Reply:
ah, that's a really good point, you're right. Ignore this request then :) we may actually need to totally remove `getOptions` to support multiple instances

Reply:
Yeah, I'll look into that too